src/applications/edu-benefits/0994/pages/trainingProgramsInformation.js

```diff
@@ -45,23 +46,34 @@ export const uiSchema = {
           },
         },
       },
-      location: {
+      'view:location': {
```

Comment:
It should work the way you had it, I'll open a bug for us to look into why it isn't.